kymatio/scattering1d/frontend/base_frontend.py

```diff
@@ -81,6 +81,7 @@ def create_filters(self):
         self.phi_f, self.psi1_f, self.psi2_f = scattering_filter_factory(
             self.J_pad, self.J, self.Q, self.T,
             r_psi=self.r_psi, sigma0=self.sigma0, alpha=self.alpha)
+        ScatteringBase._check_filterbanks(self.psi1_f, self.psi2_f)
```

Comment:
Why not move this into scattering_filter_factory?

Reply:
because the scattering filter factory is dimension-specific. the `checks_filterbanks` routine could perhaps serve 2D and 3D code too (i think)

Reply:
because the scattering filter factory is dimension-specific. the `checks_filterbanks` routine could perhaps serve 2D and 3D code too (i think)